Doc/whatsnew/3.15.rst

```diff
@@ -263,6 +263,14 @@ http.client
   (Contributed by Alexander Enrique Urieles Nieto in :gh:`131724`.)
 
 
+http.cookies
+------------
+
+* Allow '``"``' double quotes in cookie values with a more lenient regex for
+  double-quoted strings.
```

Comment:
```suggestion
* Allow '``"``' double quotes in cookie values with a more lenient regex for
  double-quoted strings by not explicitly excluding a quote inside the value part of a cookie.
```

It's pretty similar to what was there before, just doesn't exclude the quotes: https://github.com/python/cpython/pull/113663/files#diff-e49aab1421911b035542d1a9221c31b8c74053f0e8b28f90f03af66177026176R429

Reply:
I am good with the shorter version too. It is clear without confusing on the internal details like lenient regex (which the reader of whatsnew doc doesn't need to bother about) and if they really want they could refer to git log, bug report and other discussions.